wordpress/wp-content/plugins/cds-base/classes/Modules/ListManager/app/public/index.html

```diff
@@ -8,6 +8,6 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="list-manager-app"></div>
+    <div id="list-manager-app" data-ids='[{"name":"List One","service_id":"a7902fc7-37f0-419c-84c8-3ab499ee24c8"},{"name":"Service 222","service_id":"9e7cf7e0-d40d-4e43-a3c7-ea84344f4291"}]'></div>
```

Comment:
Was this left in by accident?

Reply:
That's hardcoded for the standalone version --- you can spin up React on localhost:3000 outside of WP.